python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1735,8 +1735,14 @@ def get_run_ids(
         filters: Optional[RunsFilter] = None,
         cursor: Optional[str] = None,
         limit: Optional[int] = None,
+        prioritized: bool = False,
     ) -> Sequence[str]:
-        return self._run_storage.get_run_ids(filters, cursor=cursor, limit=limit)
+        return self._run_storage.get_run_ids(
```

Comment:
`cursor` won't work with `prioritized` right? probably worth actively erroring on non-None `cursor` and `prioritized` `True`

Reply:
Hm. I think it should still work insofar as it'll only prioritize runs after that cursor. I think you're right that it's unlikely anybody actually wants to call it that way.